.github/workflows/PublishTestResults.yml

```diff
@@ -0,0 +1,54 @@
+# ==============================================================================
+# Authors:
+#   Patrick Lehmann
+#   Unai Martinez-Corral
+#
+# ==============================================================================
+# Copyright 2020-2021 The pyTooling Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ==============================================================================
+name: Publish Unit Test Results
+
+on:
+  workflow_call:
+
+jobs:
+  PublishTestResults:
+    name: 📊 Publish Test Results to Issue Comment
+    runs-on: ubuntu-latest
+    if: always()
+
+    steps:
+      - name: ⏬ Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: artifacts
```

Comment:
For performance reasons, it would be desirable to specify which artifact to download. That can be problematic, indeed, because users might want to upload multiple artifacts but not all of them. Therefore, for now, we can keep this as-is, and filter it when calling the Action below.

Reply:
I only found a way to download one single artifact or all of them.
There seems to be no way to filter for `UnitTesting-*`.

We need a way to publish multiple xml files at once or merge them. But even with a previous merge step, we need to be able to specify some artifacts pattern.

Reply:
Yes. Unfortunately, actions/download-artifact does not allow downloading some artifacts only. We would need an alternative Action for that purpose. That's why having all of them downloaded is acceptable for now.